splinterd/src/daemon.rs

```diff
@@ -382,6 +388,24 @@ impl SplinterDaemon {
             rest_api_builder = rest_api_builder.add_resources(circuit_resource.resources());
         }
 
+        #[cfg(feature = "biome")]
+        {
```

Comment:
When we ever remove this feature, we would have to change the indent of the implementation (or leave a block indented without purpose). Maybe consider making this a function as a result, and calling the function. The call to the function and the function itself can both be guarded by the feature.

Reply:
Update with the add_biome_routes function. It's a little gross with the memory swapping but it works